plugins/oracledb_metrics.yaml

```diff
@@ -0,0 +1,649 @@
+                  FROM (SELECT (SELECT DBID FROM SYS.GV_$$DATABASE) DATABASE_ID, (SELECT GLOBAL_NAME FROM sys.GLOBAL_NAME) GLOBAL_NAME, INST_ID, CURSORS_CUMULATIVE, CURSORS_CURRENT, SORTS_MEM, SORTS_DISK, SORTS_ROWS, READ_TOTAL, WRITE_TOTAL, READ_TOTAL_BY, WRITE_TOTAL_BY, LOGONS_CURRENT, CLIENT_RECV_BY, DBLINK_RECV_BY, CLIENT_SENT_BY, DBLINK_SENT_BY, LOGONS_CUMULATIVE, USER_CALLS, USER_COMMITS, USER_ROLLBACKS 
+                  FROM (SELECT * FROM SYS.GV_$$sysstat
+                    WHERE NAME IN ('opened cursors cumulative', 'opened cursors current', 'sorts (memory)', 'sorts (disk)', 'sorts (rows)', 'physical read total IO requests', 'physical write total IO requests', 'physical read total bytes', 'physical write total bytes', 'logons current', 'bytes received via SQL*Net from client', 'bytes received via SQL*Net from dblink', 'bytes sent via SQL*Net to client', 'bytes sent via SQL*Net to dblink', 'logons cumulative', 'user calls', 'user commits', 'user rollbacks')
+                    )
+                    PIVOT(
+                      MAX(VALUE)
+                      FOR NAME
+                      IN (
+                        'opened cursors cumulative' CURSORS_CUMULATIVE,
+                        'opened cursors current' CURSORS_CURRENT,
+                        'logons cumulative' LOGONS_CUMULATIVE,
+                        'logons current' LOGONS_CURRENT,
+                        'sorts (memory)' SORTS_MEM,
+                        'sorts (disk)' SORTS_DISK,
+                        'sorts (rows)' SORTS_ROWS,
+                        'physical read total IO requests' READ_TOTAL,
+                        'physical write total IO requests' WRITE_TOTAL,
+                        'physical read total bytes' READ_TOTAL_BY,
+                        'physical write total bytes' WRITE_TOTAL_BY,
+                        'bytes received via SQL*Net from client' CLIENT_RECV_BY,
+                        'bytes received via SQL*Net from dblink' DBLINK_RECV_BY,
+                        'bytes sent via SQL*Net to client' CLIENT_SENT_BY,
+                        'bytes sent via SQL*Net to dblink' DBLINK_SENT_BY,
+                        'user calls' USER_CALLS,
+                        'user commits' USER_COMMITS,
+                        'user rollbacks' USER_ROLLBACKS
+                      )
+                    )
+                  )
+                  GROUP BY DATABASE_ID, GLOBAL_NAME, INST_ID
+  service:
+    pipelines:
+      metrics/oracledb:
+        receivers: [sqlquery/oracledb]
```

Comment:
ops-agent adds a processor at the end to rename the uppercase labels to lowercase; Do we want that behavior in our collector or are we OK with the all-caps?

Reply:
Yeah I omitted that on purpose because it was ops-agent specific behavior. I am wondering though, probably is worth adding that processor

Reply:
@BinaryFissionGames added mind just doing a double check on the new processor addition?